frontend/src/components/Dashboard/Main/Carousel.jsx

```diff
@@ -76,13 +76,13 @@ class Carousel extends Component {
                         }}/>
                 }
 
-                Your Top Track{this.props.topTracks !== null && this.props.topTracks !== undefined && this.props.topTracks.length !== 0 ? this.props.topTracks.items.length > 1 ? 's' : '' : ''}:
+                Your Top Track{this.props.topTracks && this.props.topTracks.length !== 0 ? this.props.topTracks.length > 1 ? 's' : '' : ''}:
             </h2>
             <div className={`${styles.tracksWrapper}`}>
                 {
-                    this.props.topTracks !== null && this.props.topTracks !== undefined
+                    this.props.topTracks
                         ? this.props.topTracks.length !== 0
```

Comment:
this can probably be chained with && too

Reply:
I hate these ifs